src/components/profile/profileNav/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { BottomNavigation, BottomNavigationAction } from '@mui/material';
+
+interface Props {
+  value: string | number;
+  navigationData: { label: number; title: string }[];
+  onChange:
+    | ((event: React.SyntheticEvent<Element, Event>, value: string) => void)
+    | undefined;
+}
+const ProfileNav = ({ value, navigationData, onChange }: Props) => {
+  return (
+    <BottomNavigation value={value} onChange={onChange} showLabels>
+      {navigationData.map(({ label, title }) => (
+        <BottomNavigationAction label={label} icon={title} />
```

Comment:
```suggestion
        <BottomNavigationAction label={label} icon={title} key={title} />
```
map에 key 값 넣어주면 좋을 것 같습니다!